src/names.c

```diff
@@ -110,6 +110,14 @@ typedef struct _lazy_sym {
    void       *ctx;
 } lazy_sym_t;
 
+typedef struct _label_cnts label_cnts_t;
```

Comment:
You don't need to forward-declare the struct if it doesn't have any self-references. You can just declare it like

```c
typedef struct { .... } label_cnts_t;
```

Reply:
I removed it, srry. I am HW engineer so its been some time since I did some more intense C than for low-level CPU test in SoC RTL simulation... Therefore I loose the nuances :)